bibliopixel/remote/control.py

```diff
@@ -0,0 +1,140 @@
+import multiprocessing
+import inspect
+from .. animation import BaseAnimation
+from .. animation.animation import COMPLETE_REASON
+from . import server
+from .. import log
+
+
+DEFAULT_OFF = 'OFF_ANIM'
+DEFAULT_ANIM_CONFIG = {
+    'bgcolor': '#00ff00',
+    'font_color': '#ffffff',
+    'display': None
+}
+
+
+class RemoteControl():
+    def __init__(self, base_config, animations):
+        # These start with the defaults
+        self.config = {
```

Comment:
Extract this dictionary as an external constant.  Why are there two different sorts of default animation configurations?